### DISCUSSION TITLE
Setting the Basic Normalized tabular data via the API

### DISCUSSION THREAD

**Discussion by cad2040** [airbytehq/airbyte#34729] [Connector Questions]
Good morning,

I have been developing a method of deploying sources, destinations and connections via DevOps using the Airbyte API. I have it working pretty well, but notice normalization is always set to Raw Data (JSON) upon deployment. I have hunted through the API docs here https://reference.airbyte.com/reference/start but can't seem to locate a way to alter this setting via the API currently. I wanted to reach out to see if I have missed anything, or if this just isn't possible at the moment.

**Comment by sajarin** [comment]
You're correct that the Airbyte API documentation does not explicitly mention how to enable the "Normalized tabular data" option when creating a connection. However, a user on the Airbyte forum found a workaround by inspecting the network tab with the UI. They used a different endpoint, /api/v1/web_backend/connections/create, and populated the operations field as follows:

```
operations: [
  {
    name: 'Normalization',
    workspaceId: 'YOUR_WORKSPACE_ID',
    operatorConfiguration: {
      operatorType: 'normalization',
      normalization: {
        option: 'basic'
      }
    }
  }
],
```
You can find this information in this [forum post](https://discuss.airbyte.io/t/4002).

Please note that this is not officially documented and might not work as expected. The officially documented way to enable "Normalized tabular data" is through the Airbyte UI during the connection setup process, as mentioned in the [Airbyte documentation](https://docs.airbyte.com/using-airbyte/core-concepts/basic-normalization#destinations-that-support-basic-normalization).

I hope this helps! If you have any other questions, feel free to ask.

[a comment by an automated account is not shown]
